Source/AtomicEditor/Application/AEEditorCommon.cpp

```diff
@@ -255,4 +258,53 @@ bool AEEditorCommon::ReadPreferences()
     return true;
 }
 
+void AEEditorCommon::ValidateWindow()
+{
+    Graphics* graphics = GetSubsystem<Graphics>();
+    IntVector2 windowPosition = graphics->GetWindowPosition();
+    int monitors = graphics->GetMonitorsNumber();
```

Comment:
Ok, let's fix "GetMonitorsNumber" from a previous PR, this should be: GetNumMonitors() (which is short for get number of monitors, sounds better in english)